### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by pythonhubdev** [nank1ro/flutter-shadcn-ui#10] [General]
Can we please create a discord for this I am interested in Contributing to the UI and would be happy to collaborate and work. I went through `theme` discussion couldn't get much of the context since I haven't gone through the source code completely yet. Can we do this? @nank1ro

**Comment by dickermoshe** [comment]
Currently discussion are being kinda fractured between here & Twitter.
Adding another place may centralize discussions or further fragment them.

**Reply by pythonhubdev** [reply]
I think we can just mention the community is on Discord so people will automatically join there and also we can add a post on twitter.

**Comment by nank1ro** [comment]
Why would you prefer a Discord server over the Discussions here on GitHub?
Here everything would be more organised by topic, whereas on Discord it is more of a chat.

**Reply by pythonhubdev** [reply]
Got it because like discord is easy for communication and it also has whole lot of other features as well it's upto you was a suggestions just to make communication easy.

**Comment by nank1ro** [comment]
Added the Discord server
[Invitation link](https://discord.gg/ZhRMAPNh5Y)
